src/main/java/me/totalfreedom/totalfreedommod/command/Command_loginmessage.java

```diff
@@ -24,11 +24,10 @@ public boolean run(final CommandSender sender, final Player playerSender, Comman
 
         if (args.length == 0)
         {
-            playerSender.openInventory(plugin.sh.generateLoginMessageGUI(playerSender));
+            msg("You must set a valid login message! Usage: /<command> <message>", ChatColor.RED);
```

Comment:
See my previous post - `/loginmessage` opens up the login message GUI, allowing people to choose the available login messages. All you did was remove the ability to do so and force people to set their own.